Framework/Algorithms/src/ConvertToConstantL2.cpp

```diff
@@ -104,7 +114,9 @@ void ConvertToConstantL2::exec() {
     oldPos.getSpherical(r, theta, phi);
     V3D newPos;
     newPos.spherical(m_l2, theta, phi);
-    outputDetInfo.setPosition(i, newPos);
+
+    const size_t detIndex = inputSpecInfo.detector(i).index();
```

Comment:
How do I access `SpectrumInfo::spectrumDefinition` though? Is there something I need to import for it?